### PR TITLE
Mini Ada Game Engine - v0.1.1 [renamed from GNAT Game]

### PR DESCRIPTION
**Pull request by leogermond**

### CODE REVIEW

**Comment by mosteo** [comment]
The Arch issue is likely caused by a too old Docker image. Stand by while I look into it.

File: index/gn/gnat_game/gnat_game-0.1.0-dev.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name = "gnat_game"
+description = "A very simple game engine written in Ada"
+version = "0.1.0-dev"
```
Comment (mosteo) [inline index/gn/gnat_game/gnat_game-0.1.0-dev.toml]: I presume you want just `0.1.0` without `-dev` for publication.

**Review by mosteo — CHANGES_REQUESTED** [review]

File: index/gn/gnat_game/gnat_game-0.1.0-dev.toml (hunk 1 above)
Comment (leogermond) [reply index/gn/gnat_game/gnat_game-0.1.0-dev.toml]: Ah yes good catch, sorry I'm still learning

Comment (leogermond) [reply index/gn/gnat_game/gnat_game-0.1.0-dev.toml]: Fixed, and I fixed the name to be more neutral, as MAGE

**Comment by mosteo** [comment]
Nice name.

**Review by mosteo — APPROVED** [review]

**Pull request merged**